src/crypto/bcrypt.cr

```diff
@@ -47,13 +47,28 @@ class Crypto::Bcrypt
     0x64657253, 0x63727944, 0x6f756274,
   )
 
+  # Hashes a password using Bcrypt.
```

Comment:
Since you mentioned the salt value should be visible, why not note `Random::Secure.random_bytes(SALT_SIZE)` in the doc comment?

Reply:
I don't know... Was it the purpose of this method, abstracting this from the user?
That was one advantage of having `salt = Random::Secure.random_bytes(SALT_SIZE)` in the method signature initially.